implementation/tracker.m

```diff
@@ -204,17 +211,17 @@
 
 % Initialize and allocate
 prior_weights = zeros(params.nSamples,1, 'single');
-sample_weights = cast(prior_weights, 'like', params.data_type);
+sample_weights = cast(prior_weights, class(params.data_type));
 samplesf = cell(1, 1, num_feature_blocks);
 if params.use_gpu
     % In the GPU version, the data is stored in a more normal way since we
     % dont have to use mtimesx.
     for k = 1:num_feature_blocks
-        samplesf{k} = zeros(filter_sz(k,1),(filter_sz(k,2)+1)/2,sample_dim(k),params.nSamples, 'like', params.data_type_complex);
+        samplesf{k} = complex(zeros(filter_sz(k,1),(filter_sz(k,2)+1)/2,sample_dim(k),params.nSamples))
     end
 else
     for k = 1:num_feature_blocks
-        samplesf{k} = zeros(params.nSamples,sample_dim(k),filter_sz(k,1),(filter_sz(k,2)+1)/2, 'like', params.data_type_complex);
+        samplesf{k} = complex(zeros(params.nSamples,sample_dim(k),filter_sz(k,1),(filter_sz(k,2)+1)/2), 0);
```

Comment:
Need to add `class(params.data_type)`. The second argument in complex can be removed.